src/ome_zarr_models/v04/axes.py

```diff
@@ -17,4 +17,7 @@ class Axis(Base):
 
     name: str
     type: str | None = None
-    unit: str | None = None
+    unit: str | Any | None = None
+
+
+Axes = list[Axis]
```

Comment:
Do we want to restrict this type to `list[Axis]`? I think that will result in functions like `_ensure_axis_length` failing for tuples

Reply:
Ooh good point, changed to `Sequence`

Reply:
we could even think about making it an iterable at some point, but `Sequence` is fine for now